templates/gmcq.hbs

```diff
@@ -7,6 +7,9 @@
             <label aria-hidden="true" id="{{../_id}}-{{@index}}-aria" for="{{../_id}}-{{@index}}" class="{{#unless ../_isEnabled}}disabled {{/unless}} component-item-color component-item-text-color component-item-border {{#if _isSelected}}selected{{/if}} a11y-ignore" tabindex="-1">
 
                 <img src="{{_graphic.src}}" data-large="{{_graphic.large}}" data-small="{{_graphic.small}}"/>
+                {{#if _graphic.attribution}}
+                    <div class="gmcq-item-graphic-attribution">{{{_graphic.attribution}}}</div>
```

Comment:
Would prefer to have a generic class on attributions for ease of override styling, i.e.
```html
<div class="graphic-attribution">{{{_graphic.attribution}}}</div>
```
You could still implement component-specific styles if necessary by using selectors like
```less
.gmcq .graphic-attribution {
    // styles
}
```